### PR TITLE
Update python docker images

### PR DESCRIPTION
**Pull request by tammojan**
These are the docker images that I just used for deploying python-casacore 3.5.

### CODE REVIEW

File: docker/py310_wheel.docker [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+WORKDIR /build/sofa/20150209_a/f77/src
+RUN make -j${THREADS}
+
+
+# setup boost
+WORKDIR /build
+RUN tar jxvf /tmp/boost.tar.bz2
+WORKDIR /build/boost_${BOOST_}
+# patch for python 3.10 (multiple digits)
+#sed -i 's/(\[0-9\])/([0-9][0-9]?)/' tools/boost_install/BoostConfig.cmake
```
Comment (mnijhuis-tos) [inline docker/py310_wheel.docker:48]: This patch line is commented out. Do we still need it? Or does using BOOST 1.79 instead of 1.75 already address this issue?

File: docker/py310_wheel.docker [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    link=static,shared install
+
+# casacore wants numpy
+RUN /opt/python/${TARGET}/bin/pip install numpy==1.22.4
+
+# set up casacore
+ADD . /code
+RUN useradd -ms /bin/bash casacore
+RUN chown casacore.casacore /code
+RUN mkdir /code/build
```
Comment (mnijhuis-tos) [inline docker/py310_wheel.docker:65]: The existing py3x_wheel.docker files have `USER casacore` above this line. Why did you remove it for python3.10?

**Review by mnijhuis-tos — CHANGES_REQUESTED** [review]

File: docker/py310_wheel.docker (hunk 1 above)
Comment (tammojan) [reply docker/py310_wheel.docker]: I think indeed a boost upgrade did the trick. Building the wheels now, let's see.

File: docker/py310_wheel.docker (hunk 2 above)
Comment (tammojan) [reply docker/py310_wheel.docker:64]: I removed it because sometimes it's annoying to not be root. But for consistency I put the line back, thanks.

**Review by mnijhuis-tos — APPROVED** [review]
